{
  "info": {
    "author": "v1s1t0r999",
    "author_email": "aditya.funs.11@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "License :: OSI Approved :: MIT License",
      "Programming Language :: Python :: 3",
      "Programming Language :: Python :: 3.7"
    ],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/v1s1t0r999/LogError",
    "keywords": "",
    "license": "MIT",
    "maintainer": "",
    "maintainer_email": "",
    "name": "LogError",
    "package_url": "https://pypi.org/project/LogError/",
    "platform": "",
    "project_url": "https://pypi.org/project/LogError/",
    "project_urls": {
      "Homepage": "https://github.com/v1s1t0r999/LogError"
    },
    "release_url": "https://pypi.org/project/LogError/1.0.2/",
    "requires_dist": null,
    "requires_python": ">=3.6",
    "summary": "UNKNOWN TRACEBACKS!!!! Deal with them Carefully :-D",
    "version": "1.0.2",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 10499808,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "252b4a0700eb528ecaedf1a86b98bbe91a6f2555a451cddefd7e0ae89a693372",
        "md5": "269c21fb4802f138f823a2aa259d6fd9",
        "sha256": "decb1c226f67521dea0af6eab8fdb91e24e32a7fb8dc74a94bc7018d86f96c05"
      },
      "downloads": -1,
      "filename": "LogError-1.0.2-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "269c21fb4802f138f823a2aa259d6fd9",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": ">=3.6",
      "size": 2838,
      "upload_time": "2021-05-28T14:06:31",
      "upload_time_iso_8601": "2021-05-28T14:06:31.999865Z",
      "url": "https://files.pythonhosted.org/packages/25/2b/4a0700eb528ecaedf1a86b98bbe91a6f2555a451cddefd7e0ae89a693372/LogError-1.0.2-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "38f8dfc0252c45410e0c05d03b4edbb4775b1f96e5fedb90b6aff265aaaf64dc",
        "md5": "e09e2f902c2d3ea1de7e93baa63b5395",
        "sha256": "945c9807ede8a80c21e879f58ab9b71b0db7fda65cb0acc2b73d6422984a21cc"
      },
      "downloads": -1,
      "filename": "LogError-1.0.2.tar.gz",
      "has_sig": false,
      "md5_digest": "e09e2f902c2d3ea1de7e93baa63b5395",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": ">=3.6",
      "size": 1428,
      "upload_time": "2021-05-28T14:06:33",
      "upload_time_iso_8601": "2021-05-28T14:06:33.693548Z",
      "url": "https://files.pythonhosted.org/packages/38/f8/dfc0252c45410e0c05d03b4edbb4775b1f96e5fedb90b6aff265aaaf64dc/LogError-1.0.2.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}